src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Security/UseContainerLevelAccessPolicy.cs

```diff
@@ -137,8 +137,7 @@ public override void Initialize(AnalysisContext context)
 
                                 if (argumentOperation != null)
                                 {
-                                    var cfg = invocationOperation.GetTopmostParentBlock()?.GetEnclosingControlFlowGraph();
-                                    if (cfg != null)
+                                    if (invocationOperation.TryGetEnclosingControlFlowGraph(out var cfg))
```

Comment:
@dotpaul We may want to back port the product changes to 2.9.x branch. You probably want to add unit tests for the two security analyzers changed here as they would have not worked in presence of property/field initializers prior to this change. I'll let you make a call if this is a critical enough fix that you'd prefer to backport, especially given you are probably better suited to write corresponding regression tests for security analyzers.

Reply:
I'll try to write a couple of tests tonight.

Reply:
#4109

Reply:
@mavasani As long as it's straightforward, I'd prefer any DFA improvements to be backported to 2.9.x. If it's not straightforward, I guess it depends. 😃